packages/nvidia_nat_core/src/nat/front_ends/fastapi/fastapi_front_end_plugin_worker.py

```
@@ -117,17 +76,16 @@ def __init__(self, config: Config):
-                    "NAT_JOB_STORE_DB_URL must be set when using Dask (configure a persistent JobStore database).")
-
             try:
+                from nat.front_ends.fastapi.async_jobs.job_store import JobStore
+                if self._db_url is None:
+                    raise RuntimeError(
+                        "NAT_JOB_STORE_DB_URL must be set when using Dask (configure a persistent JobStore database).")
                 self._job_store = JobStore(scheduler_address=self._scheduler_address, db_url=self._db_url)
                 self._dask_available = True
                 logger.debug("Connected to Dask scheduler at %s", self._scheduler_address)
+            except ImportError:
+                raise RuntimeError("Dask is not available, please install it to use the FastAPI front end with Dask.")
```

Ruff (0.15.0)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)